src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
```

Comment:
This file needs to be in Readarr.Api.V1 folder

Reply:
incoming commit